Refresh.GameServer/Types/OAuth/GitHub/GitHubApiUserResponse.cs

```diff
@@ -0,0 +1,112 @@
+namespace Refresh.GameServer.Types.OAuth.GitHub;
+
+#nullable disable
+
+public class GitHubApiUserResponse
```

Comment:
This whole class is a mess to look at... why not just set snake case and set `NullValueHandling.Ignore` globally for this? This also doesn't seem to mark nullability properly for non-value types...